.eslintrc.yml

```diff
@@ -138,6 +138,20 @@ rules:
   new-cap: ['error', { 'capIsNew': true }]
   import/order: ['error', { 'newlines-between': 'never' }]
 
+  # Account for destructuring responses from upstream services,
+  # many of which do not follow camelcase
+  # Based on original rule configuration from eslint-config-standard
+  camelcase:
+    [
+      'error',
+      {
+        ignoreDestructuring: true,
+        properties: 'never',
+        ignoreGlobals: true,
+        allow: ['^UNSAFE_'],
+      },
+    ]
+
```

Comment:
eslint-config-standard applied a lot of configuration to this rule, which had to matched for parity in order to incorporate `ignoreDestructuring`

https://github.com/standard/eslint-config-standard/blob/8c5203f7e48177345a7aaf212cb0cb5915880096/eslintrc.json#L39-L43